autofill/frame-parent.htmk

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
@jonathanKingston This was in the branch already, but I suppose it can be deleted?

Reply:
Looks like something we should drop to me.